nixos/modules/virtualisation/docker.nix

```diff
@@ -157,6 +157,7 @@ in
 
       systemd.services.docker = {
         wantedBy = optional cfg.enableOnBoot "multi-user.target";
+        after = [ "network.target" "docker.socket" ];
```

Comment:
docker-ce unit:

```
[Unit]                                                                                                                                                                   [0/0]Description=Docker Application Container Engine
Documentation=https://docs.docker.com
After=network-online.target firewalld.service containerd.service
Wants=network-online.target
Requires=docker.socket containerd.service
```

Reply:
We don't have firewalld. We do have containerd.service but I currently fail to see how docker is relating to containerd. We built docker with its own containerd.

Reply:
I would only switch from `network.target` to `network-online.target` if we have very good reason to. Right now I would say it is enough to have a working firewall and systemd / scripted networking not interfering with dockers network setup.

Reply:
Docker is the glue to get the containers in containerd running.